addons/website_sale/views/snippets.xml

```diff
@@ -95,6 +95,14 @@
                 <we-button data-change-sequence="bottom">Push to bottom</we-button>
             </div>
         </div>
+        <div data-selector="#wrapwrap > header"
+            data-no-check="true"
+            groups="website.group_website_designer">
+            <we-checkbox string="Hide Cart If Empty"
```

Comment:
Could this be renamed with "Show" ? "Always show cart" maybe? to check with SBU

Reply:
@sbu ? ;)

Reply:
I made the change with "Always show cart"

Reply:
Note that you pinged a random guy on the internet with @ sbu ^^